components/profile/contactInfoTable/__snapshots__/index.test.jsx.snap

```diff
@@ -126,7 +126,7 @@ exports[`ContactInfoTable renders an editable table of contact info 1`] = `
     >
       <input
         class="PodBrowser-input"
-        data-testid="profile-address-new-value"
+        data-testid="profile-undefined-new-value"
```

Comment:
Looks like the tests need to pass a `typestr` so these values are rendered properly.